grafana-plugin/src/containers/AlertRules/parts/index.tsx

```diff
@@ -19,13 +20,19 @@ export const ChatOpsConnectors = (props: ChatOpsConnectorsProps) => {
   const { channelFilterId, showLineNumber = true } = props;
 
   const store = useStore();
-  const { telegramChannelStore, organizationStore } = store;
+  const { organizationStore, telegramChannelStore, msteamsChannelStore } = store;
 
   const isSlackInstalled = Boolean(organizationStore.currentOrganization?.slack_team_identity);
   const isTelegramInstalled =
     store.hasFeature(AppFeature.Telegram) && telegramChannelStore.currentTeamToTelegramChannel?.length > 0;
 
-  if (!isSlackInstalled && !isTelegramInstalled) {
+  useEffect(() => {
+    msteamsChannelStore.updateMSTeamsChannels();
+  }, []);
+
+  const isMSTeamsInstalled = Boolean(msteamsChannelStore.currentTeamToMSTeamsChannel?.length > 0);
```

Comment:
`Boolean()` coercion is redundant here because `msteamsChannelStore.currentTeamToMSTeamsChannel?.length > 0` always returns `true` or `false` anyway